src/app/models/contracts-result.ts

```diff
@@ -0,0 +1,5 @@
+interface ContractsResult<T> {
```

Comment:
it is a general RestResponse model from ethparser, create a class instead of an interface